File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
A, B = map(int, input().split(' '))
C = int(input())
h = int((A * 60 + B + C) / 60)
if(h > 23):
    h = h-24
m = int((A * 60 + B + C) % 60)
print(h, m)

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let a = parseInt(input[0].split(" ")[0]);
let b = parseInt(input[0].split(" ")[1]);
let c = parseInt(input[1]);

h = Math.floor((a * 60 + b + c) / 60);
m = (a * 60 + b + c) % 60;
if (h >= 24) {
  h -= 24;
}

console.log(h, m);